RELEASE.md

```diff
@@ -101,19 +102,13 @@ $ git fetch upstream
 $ git checkout vX.Y.0
 $ git checkout -b release/vX.Y.1
 $ git cherry-pick <...>
-# Edit files, updating from `X.Y.0` to `X.Y.1`, and git add them. Currently the list is:
 #
-# * authorization-codegen/protoc-gen-contextual-auth-wiring
-# * authorization-codegen/protoc-gen-service-auth-wiring
-# * buildSrc/src/main/groovy/io.deephaven.common-conventions.gradle
-# * py/client-ticking/README.md
-# * py/client-ticking/setup.py
-# * py/client/README.md
-# * py/client/pydeephaven/__init__.py
-# * py/client/setup.py
-# * py/embedded-server/deephaven_server/__init__.py
-# * py/server/deephaven/__init__.py
-# * R/rdeephaven/DESCRIPTION
+# Edit files, updating from `X.Y` to `X.(Y+1)`
```

Comment:
This is better "# Edit files, updating from `X.Y.0` to `X.Y.1`, and git add them."  because it fits with what has already been said above, and it also tells you to "git add"